Tasks/CopyFilesToMachines/CopyFilesHelper.ps1

```diff
@@ -11,7 +11,8 @@ function DoComplete-ResourceOperation
     $logs = New-Object 'System.Collections.Generic.List[System.Object]'         
     $resourceOperationLog = New-OperationLog -Content $log
     $logs.Add($resourceOperationLog)
-
+    
+    import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```

Comment:
We are importing this module in the CopyFileToMachines.ps1 and then executing this function in this script which invokes cmdlet Complete-ResourceOperation. I think this would not be needed here as it is already being imported in the script which calls it.